coordinator/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "birdseed-escrow"
+name = "birdseed_escrow"
 version = "0.1.0"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 nostr-sdk = { version = "0.32.0", features = ["nip04"] }
-cdk = { git = "https://github.com/cashubtc/cdk" }
+cdk = "0.1.1"
```

Comment:
Great to see cdk on the rust repo now :)

Reply:
I noticed this because we were taking latest and something suddenly changed.